docs/api.md

```diff
@@ -182,6 +220,70 @@ Example Request Body:
 Returns:
 `201` if successful
 
+### POST /v2/orders (PERPETUAL)
```

Comment:
This should not be a new header. It uses the same route so should use the same `POST /v2/orders` header. There shouldn't be different ones for solo vs perpetual.

The only thing that is different is the `order` field. Make it clear that that's all that's different and refer to your order fields sections above